dandi/cli/cmd_validate.py

```diff
@@ -9,56 +9,6 @@
 from ..validate_types import Severity
 
 
-@click.command()
-@click.option(
-    "--schema", help="Validate against new BIDS schema version.", metavar="VERSION"
-)
-@click.option(
-    "--report-path",
-    help="Write report under path, this option implies `--report/-r`.",
-)
-@click.option(
-    "--report",
-    "-r",
-    is_flag=True,
-    help="Whether to write a report under a unique path in the DANDI log directory.",
-)
-@click.option(
-    "--grouping",
-    "-g",
-    help="How to group error/warning reporting.",
-    type=click.Choice(["none", "path"], case_sensitive=False),
-    default="none",
-)
-@click.argument("paths", nargs=-1, type=click.Path(exists=True, dir_okay=True))
-@map_to_click_exceptions
-def validate_bids(
-    paths,
-    schema,
-    report,
-    report_path,
-    grouping="none",
-):
-    """Validate BIDS paths.
-
-    Notes
-    -----
-    Used from bash, eg:
-        dandi validate-bids /my/path
-    """
-
-    from ..validate import validate_bids as validate_bids_
-
-    validator_result = validate_bids_(  # Controller
-        *paths,
-        report=report,
-        report_path=report_path,
-        schema_version=schema,
-    )
-
-    _process_issues(validator_result, grouping)
```

Comment:
Let's do proper deprecation cycle, to assist users in not requiring to check CHANGELOG which would not tell much about disappearance of `bids-validate` command (besides that it is merged):
- here if invoked issue a DeprecationWarning alerting users to use `validate` command instead
- if `report` option is provided, say that there is no such option any longer -- people would need to just redirect output of validator to a file if desire to store it etc.

and then invoke the "merged" `validate` itself with corresponding options.

Reply:
@yarikoptic tried to bring it back in [60f7f97](https://github.com/dandi/dandi-cli/pull/1203/commits/60f7f972487317e3c45a448b4db09a23a0046578) . For some reason it doesn't work. I think Python might be getting confused and think that `validate` is a module:

```
(dev) [deco]~/src/dandi-cli ❱ dandi validate-bids ~/src/bids-examples/asl003/
2023-02-24 13:22:13,031 [ WARNING] A newer version (0.49.1) of dandi/dandi-cli is available. You are using 0.49.0+51.g60f7f97
(*args: Any, **kwargs: Any) -> Any
2023-02-24 13:22:13,038 [    INFO] Logs saved in /home/chymera/.cache/dandi-cli/log/20230224182212Z-16933.log
Error: Context.__init__() got an unexpected keyword argument 'grouping'
```

Though I renamed it as well, and that still happens. Again these are just a few lines but I tried a buch of different fixes. Are you sure this is worth the extra effort in figuring it out? we could just make it do nothing and issue a warning...

Reply:
@yarikoptic tried to bring it back in [60f7f97](https://github.com/dandi/dandi-cli/pull/1203/commits/60f7f972487317e3c45a448b4db09a23a0046578) . For some reason it doesn't work. I think Python might be getting confused and think that `validate` is a module:

```
(dev) [deco]~/src/dandi-cli ❱ dandi validate-bids ~/src/bids-examples/asl003/
2023-02-24 13:22:13,031 [ WARNING] A newer version (0.49.1) of dandi/dandi-cli is available. You are using 0.49.0+51.g60f7f97
(*args: Any, **kwargs: Any) -> Any
2023-02-24 13:22:13,038 [    INFO] Logs saved in /home/chymera/.cache/dandi-cli/log/20230224182212Z-16933.log
Error: Context.__init__() got an unexpected keyword argument 'grouping'
```

Though I renamed it as well, and that still happens. Again these are just a few lines but I tried a buch of different fixes. Are you sure this is worth the extra effort in figuring it out? we could just make it do nothing and issue a warning...

Reply:
ok, fixed in [06e2a35](https://github.com/dandi/dandi-cli/pull/1203/commits/06e2a354df922b2c31607e1f200dc92f2ab0cde0)